frontend/src/views/IssueDetail.vue

```diff
@@ -1171,9 +1177,50 @@ export default {
       return count > 1;
     });
 
+    const database = computed((): Database | undefined => {
+      const temp = selectedStage as unknown;
+      if (create) {
+          const stage = temp as ComputedRef<StageCreate>;        
+          const databaseId = stage.value.taskList[0].databaseId;
+          if (databaseId) {
+            return store.getters["database/databaseById"](databaseId);
+          }
+        
+          return undefined;
+      }
+      const stage = temp as ComputedRef<Stage>;      
+      return stage.value.taskList[0].database;
+    });
+
+    const instance = computed((): Instance => {
+      const temp = selectedStage as unknown;
+      if (create) {
+        // If database is available, then we derive the instance from database because we always fetch database's instance.
+        // On the other hand, instance for stage.taskList[0].instanceId might not be loaded (e.g. when creating an update schema issue)
+        if (database.value) {
+          return database.value.instance;
+        }
+        const stage = temp as ComputedRef<StageCreate>;;
+        return store.getters["instance/instanceById"](
+          stage.value.taskList[0].instanceId
+        );
+      }
+      const stage = temp as  ComputedRef<Stage>;
+      return stage.value.taskList[0].instance;
+    });
+
+    const hintForSQL = computed(() => {
+      return instance.value.engine === "SNOWFLAKE" && create
+        ? ` your are altering a snowflake database, please add <schema> before each tabel altered`
```

Comment:
Suggestion: (Use <<schema>>.<<table>> to specify a Snowflake table)

The original sentence has couple of issues:

1. Incorrect grammar, "your are", "tabel"

2. should use parenthesis to be consistent with the existing baseline hint

3. Not succinct, we have limited space here and we should reduce the user's cognitive load.

4. "please add <schema> before each tabel altered", this is also ambiguous (what does before mean?), also not accurate (since user could create table as well)